media-fonts/fontawesome/fontawesome-6.1.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit font
+
+DESCRIPTION="The iconic font"
+HOMEPAGE="https://fontawesome.com"
+SRC_URI="https://github.com/FortAwesome/Font-Awesome/archive/refs/tags/6.1.1.tar.gz -> ${P}.tar.gz"
```

Comment:
```suggestion
SRC_URI="https://github.com/FortAwesome/Font-Awesome/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
```

Reply:
The jump from v5 to v6 (as far as i can tell) only adds new icons as per the changelog:
https://fontawesome.com/v6/docs/changelog/

i've updated the suggestions, but the documentation is a bit unclear in https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds in the Note: "Using the [${PN} style variables](https://devmanual.gentoo.org/ebuild-writing/variables/) inside SRC_URI is allowed, though this is not necessarily best practice"

Thanks,